### DISCUSSION TITLE
Topic representation using LLMs

### DISCUSSION THREAD

**Discussion by MattePogra** [MaartenGr/BERTopic#1440] [Q&A] [answered]
Hi,
I was playing around Topics representation using the different methods available in the documentation. I do find extremely useful the usage of ChatGPT as a summarizer of documents/topics. I was wondering if one can use open source LLMs (such as the new Llama2) to basically perform the same task that chatGPT does within Bertopic. Is there any example, guidance that one can follow?
Thank you!

**Comment by MaartenGr** [comment, marked as answer]
You can use LangChain for that to quickly get up and running with local LLMs. You can find more in the [documentation](https://maartengr.github.io/BERTopic/getting_started/representation/llm.html#langchain).
